WTWSMS/events/WtWSMS_story_events.txt

```diff
@@ -3599,4 +3598,112 @@ narrative_event = {
 	option = { 
 		name = historical_wtwsms.6.a
 	}
-} 
\ No newline at end of file
+}
+
+# Emperor has won the Isaurian Wars
+narrative_event = {
+	id = historical_wtwsms.7
+	title = historical_wtwsms.7.name
+	desc = historical_wtwsms.7.desc
+	picture = GFX_evt_victory_arch_byzantine
+	
+	only_rulers = yes
+	only_independent = yes
+	
+	trigger = {
+		start_date = 492.1.1
+		NOT = { year = 520 } # the war should be ended by this year
+		ROOT = {
+			has_landed_title = e_byzantium
+			OR = {
+				has_landed_title = c_ephesos # Capital of Asia
+				has_landed_title = c_ikonion # Capital of Pisidia
+				any_realm_lord = {
+					has_landed_title = c_ephesos # Capital of Asia
+				}
+				any_realm_lord = {
+					has_landed_title = c_ikonion # Capital of Pisidia
+				}
+			}
+		}
+		d_isauria = {
+			holder_scope = {
+				has_truce = ROOT
+			}
+		}
+		NOT = { has_global_flag = isaurian_war_ended }
+	}
+	
+	mean_time_to_happen = {
+		days = 1
+	}
+	
+	immediate = {
+		set_global_flag = isaurian_war_ended
+		d_isauria = { holder_scope = { save_global_event_target_as = target_isaurian_leader } }
```

Comment:
Why do you set a global_event_target?

Reply:
I did that be be sure it worked :) Is normal event_target enough here (for the localisation)?

Reply:
Definitely. And below you don't need it at all, as it isn't called and the character receiving the event is your target.

Reply:
That's right, I can remove it

Reply:
Done, changed to regular event_target